docs/eksempler.md

```diff
@@ -0,0 +1,26 @@
+Det følgende er kun eksempler på hvordan brukernotifikasjoner kan tas i bruk, og verdiene i feltene må tilpasses til hver enkelt applikasjon.
+
+
+#### Minimal kafka-producer som publiserer brukernotifikasjoner på kafka
+[Dette](https://github.com/navikt/brukernotifikasjoner-demo-producer) er et mini-prosjekt som viser hvordan man kan lage en minimal kafka-producer for å publisere brukernotifikasjoner som vil vises på forsiden av DittNAV. I det du produserer et event til en kafka-topic så vil kafka “notere” seg hvilken versjon av skjemaet som den konkrete meldingen ble produsert med.
+
+URL-en til Kafkas schema-register må spesifiseres, dette gjøres i din kafka-producer-config-en f.eks. slik:
```

Comment:
Skal det stå kafka-producer-config her?

Reply:
Ja, det er riktig. Hver instans av en kafka-produent/konsument har en liste av properties, og der må det for produsent spesifiseres hvor schema-registeret finnes.

Reply:
Tenkte mer på at det kanskje skal stå "dette gjøres i din kafka-producer-config" istedenfor "dette gjøres i din din kafka-producer-config-en" :)

Reply:
Ja, selvfølgelig, nå er jeg med 😉  Nå har jeg rettet det.